config/locales/en.yml

```diff
@@ -339,6 +340,7 @@ en:
       call_log: Call Log
       change_log: Change Log
       notes: Notes
+      patient_entry: Patient Entry
```

Comment:
I think this 'menu' section was referring to the overall app menu. I'd suggest moving this to a 'data_entry' section, perhaps?

Reply:
Sounds good! I will implement that.